src/block/README.md

```diff
@@ -48,22 +48,22 @@ All other props are spread onto the base element.
   * Accepts all themeable color properties (`primary200`, etc.).
 * `font?: string`
   * Accepts all themeable font properties (`font200`, etc.).
-* `alignContent?: 'start' | 'end' | 'center' | 'between' | 'around' | 'stretch'`
-* `alignItems?: 'start' | 'end' | 'center' | 'baseline' | 'stretch'`
-* `alignSelf?: 'auto' | 'start' | 'end' | 'center' | 'baseline' | 'stretch'`
-* `flexDirection?: 'row' | 'column'`
-* `display?: 'none' | 'flex' | 'block' | 'inline-block'`
-* `flex?: 'grow' | 'shrink' | 'none'`
-* `justifyContent?: 'start' | 'end' | 'center' | 'between' | 'around'`
-* `justifySelf?: string`
-* `position?: 'static' | 'absolute' | 'relative' | 'fixed'`
+* `alignContent?: [available values](https://developer.mozilla.org/en-US/docs/Web/CSS/align-content)`
+* `alignItems?: [available values](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items)`
+* `alignSelf?: [available values](https://developer.mozilla.org/en-US/docs/Web/CSS/align-self)`
+* `flexDirection?: [available values](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction)`
+* `display?: [available values](https://developer.mozilla.org/en-US/docs/Web/CSS/display)`
+* `flex?: string`
+* `justifyContent?: [available values](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content)`
+* `justifySelf?: [available values](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-self)`
+* `position?: 'static' | 'absolute' | 'relative' | 'fixed' | 'sticky'`
 * `width?: string`
 * `minWidth?: string`
 * `maxWidth?: string`
 * `height?: string`
 * `minHeight?: string`
 * `maxHeight?: string`
-* `overflow?: 'visible' | 'hidden' | 'scroll' | 'scrollX' | 'scrollY' | 'auto'`
+* `overflow?: 'visible' | 'hidden' | 'scroll' | 'scrollX' | 'scrollY' | 'auto' | 'inherit' | 'initial' | 'unset'`
```

Comment:
Should we make it similar description with a [link to ->](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow)?

Reply:
block does not follow css convention for this property https://github.com/uber-web/baseui/blob/master/src/block/styled-components.js#L129-L135 so wanted to leave it separate